src/components/ActionButton/ActionButton.tsx

```diff
@@ -139,7 +158,7 @@ const ActionButton = ({
   return (
     <button
       className={buttonClasses}
-      disabled={disabled}
+      disabled={disabled || showLoader}
```

Comment:
```suggestion
      disabled={disabled === null ? showLoader : disabled}
```
I think this is too drastic, as it makes it impossible to enable a button if it is loading. There are use cases where we want to allow re-sending an action from a button in loading state. The current change would prevent the user from re-sending.

I suggest changing the disabled prop default to `null`. If disabled is not set by the caller, we can shadow the `showLoader` behavior. But if disabled is set by the caller, we should respect the value coming in.